BRAD/endpoints.py

```
@@ -152,20 +152,26 @@ def initiate_start():
+    ).get_agent()
     delete_dirs_without_log(initial_agent)
     log_path = initial_agent.state['config'].get('log_path')
     default_session = os.path.join(log_path, DEFAULT_SESSION_EXTN)
     set_global_output_path(log_path, default_session)
     # default agent to be used
-    default_agent = AgentFactory(tool_modules=TOOL_MODULES, 
-                                 start_path=default_session, 
-                                 interactive=False, 
-                                 persist_directory=DATABASE_FOLDER,
-                                 db_name=CACHE.get('rag_name')).get_agent()
+    default_agent = AgentFactory(
```

Ruff

Local variable `default_agent` is assigned to but never used

Remove assignment to unused variable `default_agent`

(F841)